lib/utilities/global-npm-version.js

```diff
@@ -0,0 +1,9 @@
+// Get the global npm version. You cannot require global npm modules, including
+// global npm itself, but you can still *run* global npm.
+const spawnSync = require('child_process').spawnSync;
```

Comment:
@stefanpenner - I am happy with that path also.  The original reason we added `npm` to the output of `ember version` was so that bug reports can include most of what we needed to know with simply sharing `ember version`.  This was of course before we started bundling our own, so the resulting value was a more accurate reflection of the system state.